au3/libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -547,23 +550,33 @@ size_t WaveClip::GreatestAppendBufferLen() const
 void WaveClip::OnProjectTempoChange(
    const std::optional<double>& oldTempo, double newTempo)
 {
+   const auto ratioChange = *oldTempo / newTempo;
+
    if (!mRawAudioTempo.has_value())
       // When we have tempo detection ready (either by header-file
       // read-up or signal analysis) we can use something smarter than that. In
       // the meantime, use the tempo of the project when the clip is created as
       // source tempo.
       mRawAudioTempo = oldTempo.value_or(newTempo);
 
+   if (!mStretchEnabled && !muse::is_equal(ratioChange, 0.0)) {
```

Comment:
can you explain this one please `muse::is_equal(ratioChange, 0.0)` ?

Reply:
oh... I've handled that in a wrong way... when std::nullopt is passed as oldTempo then ratioChange is 0.0... I'll fix that

Reply:
yes, exactly, my bad, fixed